src/Followable.php

```diff
@@ -21,32 +21,93 @@
  */
 trait Followable
 {
+    /**
+     * @return bool
+     */
+    public function isPrivateUser()
```

Comment:
Thanks to the great PR, But I think this method may require a more appropriate name,  for example, `isFollowerNeedToBeReviewed` or `isFollowerNeedsReview`, `isFollowerNeedsApproval`? Sorry, I'm not good at English.

Reply:
Good input! Will update this